packages/@coorpacademy-components/src/template/my-learning/index.js

```diff
@@ -154,7 +154,10 @@ const MyLearning = (props, context) => {
   const graphDatas = useMemo(
     () =>
       pipe(
-        map(skill => [skill, skillsInformation[skill].stats.score]),
+        map(skill => [
+          skill,
+          skillsInformation[skill].stats ? skillsInformation[skill].stats.score : 0
```

Comment:
```suggestion
          getOr(0, [skill, 'stats'], skillsInformation)
```

Reply:
finally you remove the getOr, there was an issue with it?

Reply:
please use it, that will avoid it to break if skill data or stats props isnt there for some entries.
(happened a lot to Adams and I while working on the backend, but not sure we supported all the cases)

Reply:
Original suggestion was wrong (partial path)
```suggestion
          getOr(0, [skill, 'stats', 'score'], skillsInformation)
```